targets/science/include/PinNames.h

```diff
@@ -164,33 +165,31 @@ typedef enum {
   LED_G = PC_2,
   LED_B = PC_3,
 
-  BUTTON_1 = PB_0,
-  BUTTON_2 = PB_1,
+  BUTTON_1 = PA_2,
 
   /**** Servo Pins *****/
-  // Felix TODO: name these better. should be SRVO_PWM_CVR for cover servo or something
-  SRVO_PWM_1 = PB_2,
-  SRVO_PWM_2 = PB_10,
+  SRVO_PWM_SHVL = PB_2,
+  SRVO_PWM_LID = PB_10,
 
   /**** Actuator Pins ****/
-  // Felix TODO: name these better. should be MTR_PWM_ELVTR for elevator motor
-  MTR_PWM_1 = PA_6,
-  MTR_DIR_1 = PA_5,
-  MTR_PWM_2 = PA_7,
-  MTR_DIR_2 = PC_4,
+  MTR_PWM_ELVTR = PA_6,
+  MTR_DIR_ELVTR = PC_5,
+  MTR_PWM_CENTRIFUGE = PA_7,
+  MTR_DIR_CENTRIFUGE = PC_4,
 
   /**** Limit Switch Pins ****/
-  // Felix TODO: name these better
-  LIM_SW_1 = PB_4,
-  LIM_SW_2 = PB_3,
-  LIM_SW_3 = PD_2,
-  LIM_SW_4 = PC_12,
+  CENTRIFUGE_SW_1 = PC_15,
+  CENTRIFUGE_SW_2 = PC_14,
+  ELVTR_SW_3 = PC_13,
+  ELVTR_SW_4 = PB_9,
+  LIM_SW_5 = PB_8,
```

Comment:
each limit switch should have the word LIM in it. also shorten centrifuge maybe? also specify what each one is for. for example you should have LIM_SW_ELVTR_DN and LIM_SW_ELEVTR_UP or something.

Also, what's lim_sw_5 for?

Reply:
I'm not sure what limit switch 5 is for. it was in the schematic so I included it

Reply:
Could you please check with EE and ME what this limit switch is for and make sure we use it properly?